docs/operations/SERVER_LIFECYCLE.md

```
@@ -0,0 +1,286 @@
+The Admin UI will provide:
+- Visual toggles for enable/disable
+- Real-time status indicators
+- One-click bulk operations
+- Configuration history
+
+## API Endpoints
+
+*Coming in Sprint 5*
+
+Planned programmatic control:
+```bash
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)